packages/grafana-toolkit/README.md

```diff
@@ -63,31 +90,23 @@ To configure Typescript create `tsconfig.json` file in the root dir of your app.
-
-### Tests
-grafana-toolkit comes with Jest as a test runner. It runs tests according to common config locted in `packages/grafana-toolkit/src/config/jest.plugin.config.ts`.
+grafana-toolkit templates comes with [default config for TSLint](https://github.com/grafana/grafana/blob/master/packages/grafana-toolkit/src/config/tslint.plugin.json). As for now there is now way to customise TSLint config.
 
-For now the config is not extendable, but our goal is to enable custom jest config via jest.config or package.json file. This might be required in the future if you want to use i.e. `enzyme-to-json` snapshots serializer. For that particular serializer we can also utilise it's API and add initialisation in the setup files (https://github.com/adriantoine/enzyme-to-json#serializer-in-unit-tests). We need to test that approach first.
+### Prettier
+When building extension with [`plugin:build`](#building-extensions) task, grafana-toolkit performs Prettier check. If the check detects any Prettier issues, the build will not pass. To avoid such situation we suggest developing plugin with [`plugin:dev --watch`](#developing-extensions) task running, that trys to fix Prettier issues automatically.
 
-#### Jest setup
-We are not opinionated about tool used for implmenting tests. Internally at Grafana we use Enzyme. If you want to configure Enzyme as a testing utility, you need to configure enzyme-adapter-react. To do so, you need to create `[YOUR_APP]/config/jest-setup.ts` file that will provide React/Enzyme setup. Simply copy following code into that file to get Enzyme working with React:
+> Only applies if you are setting up an extension not using [plugin:create](#scaffolding-extensions) task
 
-```ts
-import { configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+In order for your IDE to pickup our Prettier config you need ti ceate `.prettierrc.js` file in the root directory of your plugin with following contents:
 
-configure({ adapter: new Adapter() });
+```js
+module.exports = {
+  ...require("./node_modules/@grafana/toolkit/src/config/prettier.plugin.config.json"),
+};
 ```
 
-grafana-toolkit will use that file as Jest's setup file. You can also setup Jest with shims of your needs by creating `jest-shim.ts` file in the same directory: `[YOUR_APP]/config/jest-shim.ts`
-
-Adidtionaly, you can also provide additional Jest config via package.json file. For more details please refer to [Jest docs](https://jest-bot.github.io/jest/docs/configuration.html#verbose-boolean). Currently we support following properties:
-- [`snapshotSerializers`](https://jest-bot.github.io/jest/docs/configuration.html#snapshotserializers-array-string)
-
-
-## Working with CSS & static assets
-We support pure css, SASS and CSS in JS approach (via Emotion).
+### Working with CSS & static assets
+We support pure css, SASS and css-in-js approach (via Emotion).
 
 1. Single css/sass file
```

Comment:
Would it make sense to make these into proper headers? It's difficult to distinguish them from the rest of the text (and they're not ordered anyway, right?). 

Also, css/sass are acronyms so they should probably by uppercased.